hunts/src/HuntViewMain.js

```diff
@@ -90,8 +92,9 @@ const TABLE_COLUMNS = [
   {
     Header: "Tags/Metas",
     id: "tags",
-    accessor: (row) => row.tags.map(({ name }) => name).join(" "),
+    accessor: "tags",
```

Comment:
This will break searching by tags (eg you can search a meta name and see all puzzles with that meta because they match the tag). Two options are to either get the global filtering to work with the new `tags` object or to keep this as a string, add some kind of separator character(s) around/between the tag strings (eg put a pipe between tags or surround them with <>) so that the string search still works, and then parse that inside your filter.

Reply:
This also turned out just to not do anything.